samples/bot-task-reminder/nodejs/appPackage/manifest.json

```diff
@@ -0,0 +1,52 @@
+﻿{
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/v1.11/MicrosoftTeams.schema.json",
+  "manifestVersion": "1.11",
+  "version": "1.0.1",
+  "id": "{{Microsoft-App-Id}}",
+  "packageName": "com.microsoft.teams.aaptaskreminder",
+  "developer": {
+    "name": "Microsoft",
+    "websiteUrl": "https://www.microsoft.com",
+    "privacyUrl": "https://www.microsoft.com/privacy",
+    "termsOfUseUrl": "https://www.microsoft.com/termsofuse"
+  },
+  "name": {
+    "short": "App task reminder",
+    "full": "App task reminder"
+  },
+  "description": {
+    "short": "App task reminder",
+    "full": "This sample shows feature where user can schedule a task and get reminder on the scheduled date and time."
```

Comment:
This sample shows a feature where user can schedule a recurring task and get a reminder on the scheduled time